### ISSUE TITLE
PHP 8.1 Compatibility

### ISSUE THREAD

**Issue by kovshenin** [humanmade/altis-local-server#523]
Add support for a PHP 8.1 Docker image [when it's available](https://github.com/humanmade/docker-wordpress-php/issues/119). We should also update the symfony/yaml dependency to 6.0 or newer since it had some [related fixes](https://github.com/symfony/yaml/commit/44d76be69797377e705a48c5f53ea386c5e5b318).

Acceptance criteria:

- [x] Local Server runs PHP 8.1 image
- [ ] Symfony/yaml dependency updated
- [x] Works as expected with PHP 8.1 on the host system

**Comment by veselala** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Vulcan-6284e0dfb386b00c83981426/issues/humanmade/altis-local-server/523?planning-poker) with ZenHub @ferschubert-hm @kovshenin @mikelittle

**Comment by kovshenin** [comment]
Going to back out of upgrading the symfony/yaml dependency for now.

The problem that raised the need for updating that dependency was a PHPCS compatilibity check that reported deprecated and removed configuration directives in two files (symfony/yaml/Inline.php, symfony/yaml/Parser.php):

> INI directive 'mbstring.func_overload' is deprecated since PHP 7.2

These were fixed in the 6.1 branch of the repository, so I tried upgrading to that. However, it looks like we have a large number of dependencies, especially around altis/dev-tools (looking at you, Codeception) that have fixed requirements for symfony/yaml `< 6.0` for the versions that we use currently. I looked at upgrading some of those dependencies, however most of them look like work still in progress.

I was unable to get a working dependency tree with symfony/yaml version 6.1 without resorting to ugly hacks, and decided we can upgrade later when things upstream are a little more polished.

The warnings themselves aren't critical, `ini_get()` will simply return false in PHP 8.0 (when this directive was removed) and above, which will assume that overloading is turned off.
